### DISCUSSION TITLE
Exporting data using piccolo admin is appending `=` on each cell of csv

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [piccolo-orm/piccolo_admin#243] [General]
Today, I tried to export a table and found it is appending `=` i.e. `is equal to` sign in each cell of the csv file, which was quite annoying.
I request you to please fix it.
Thank you

**Comment by dantownsend** [comment]
This is a weird 'feature' of CSVs. You can see an explanation here:

https://stackoverflow.com/a/34596323

We can disable this behaviour in the library we use. Here's a comment explaining how:

https://github.com/jecovier/vue-json-excel/issues/166#issuecomment-1150738057

If you open the file with Excel, then these characters disappear.

I think the best way for us to solve it would be to show a popup modal which gives the user the options of having these `=` characters or not.

**Reply by sinisaos** [reply]
@dantownsend you are right, the equal sign is only on preview. If the file is opened in Excel, LibreOffice or OpenOffice, these characters disappear and I think that is ok.

![csv](https://user-images.githubusercontent.com/30960668/198836334-0643fd87-daf4-464b-90a6-64b670b989b5.png)
